### ISSUE TITLE
Support a required attribute for html5 form validation

### ISSUE THREAD

**Issue by kmturley** [indrimuska/angular-selector#13]
At the moment if you add the required attribute to angular-selector, it's not passed to the nested form field template. This mean html5 form validation does to require it to be filled in.

Consider this example:

```
<form action="/create" method="post">
    <div class="form-group">
        <label for="title">Title</label>
        <input id="title" type="text" placeholder="Enter your title" model="title" required />
    </div>
    <div class="form-group">
        <label for="tags">Tags</label>
        <select id="tags" placeholder="Select multiple" model="tags" multi="true" selector required>
            <option value="item1">Item 1</option>
            <option value="item2">Item 2</option>
            <option value="item3">Item 3</option>
        </select>
    </div>
    <div class="form-group">
        <input type="submit" value="Submit" />
    </div>
</form>
```

The title field will have html5 form validation, but angular-select will not, even though they both have a required attribute.

**Comment by indrimuska** [comment]
Yes, adding `required` attribute won't work for Angular Selector.
I will work for a patch.

**Comment by indrimuska** [comment]
FYI this is now available in [v1.2.0](https://github.com/indrimuska/angular-selector/releases/tag/1.2.0). :)

**Comment by kmturley** [comment]
amazing!! will check it out, I needed this to be able to put a red border around invalid fields
